drakcore/drakcore/frontend/src/AnalysisStatus.js

```diff
@@ -3,6 +3,54 @@ import { Component } from "react";
 import { Redirect } from "react-router-dom";
 import "./App.css";
 import api from "./api";
+import RFB from "@novnc/novnc";
+
+class PasswordField extends Component {
```

Comment:
Do we event want it to be password protected? What's the reasoning behind this?

Reply:
VNC is password protected by default, so the analysis VMs are not accessible even if they are accidentally exposed to internet/intranet.

This doesn't perfectly add up since we have a publicly available unauthenticated frontend. This has to be reconsidered on the design level. Right now I think we can leave it as it is.